bob-apps/bin/blob_recovery_tool/command.rs

```diff
@@ -401,6 +405,91 @@ impl MigrateCommand {
     }
 }
 
+pub struct GetBlobInfoCommand {
+    path: PathBuf,
+}
+
+impl GetBlobInfoCommand {
```

Comment:
This subcommand should have optional parameter `--full` to specify, that full scan of blob is required. In short mode (default one) it should print only information from BLOB header